lexicons/bsky.app/updateProfile.json

```diff
@@ -3,7 +3,6 @@
   "id": "app.bsky.updateProfile",
```

Comment:
think you missed `postNotificationsSeen` in the bsky lexicon

Reply:
fixed